lessons/pydata/pandas_types/index.ipynb

```diff
@@ -0,0 +1,6941 @@
+{
```

Comment:
<p>Nemělo by se říct, že csv i excel nejsou úplně ideální? Co doporučit Feather pro lepší výkon / škálovatelnost / přenositelnost?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/23/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='117'/>

Reply:
<p>Zmínil jsem feather a parquet.</p>